blockchain/state_manager/src/lib.rs

```diff
@@ -234,4 +243,35 @@ where
             )),
         }
     }
+
+    pub fn lookup_id<'a>(&'a self, addr: &Address, ts: &Tipset) -> Result<Address, Error> {
```

Comment:
lifetime not needed here
```suggestion
    pub fn lookup_id(&self, addr: &Address, ts: &Tipset) -> Result<Address, Error> {
```